pages/vi/vi-first-steps.md

```diff
@@ -94,11 +94,7 @@ Follow the directions under the [Nation Planet tab](vi-nation.md).
 
 ## Step 8 - Create Issues and Pull Requests
 
-Create three more issues, add comments to three other issues, and resolve three issues by making the necessary changes in your repo with three separate pull requests. You should resolve three issues created by yourself, and make sure you mention in the issue you are working on, that you are in fact working on it. **Besides, you should make sure someone from the OLE team confirms to make the change that your issue has brought up before you start working on the issue.** You will follow the same steps as in [GitHub Issues](vi-github-issues.md), continuing to improve this Markdown Wiki so that it is educational, yet easy to understand for future interns.
-
-Just as your learning with this Wiki was made possible by the efforts of previous interns, now we turn to you to continue that tradition and help future interns take their First Steps. Through this step, you should try to further improve our Markdown Wiki, with better explanations, formatting fixes, and more! You must have made your pull requests and have them each approved by at least two other OLE team members/interns in order for it to be merged to the upstream repository. Just as before, announce your Pull Request on the Gitter chat so other members can review it. You may need to make some additional corrections. Learn by teaching - or editing our teaching tool, in this case!
-
-**NOTE**: When you are fixing an issue, mention the issue number you are fixing in the title of your pull request and make a new branch for every new issue from the master branch.
+Follow the directions under the [Create Issues and Pull Requests tab](vi-create-issues-and-pull-requests.md). Create three more issues, add comments to three other issues, and resolve three issues by making the necessary changes in your repo with three separate pull requests.
```

Comment:
You can reduce this paragraph but keep two paragraph below otherwise people will not remember to send comment on chat and put issue number.